src/cloud/cdn/configure-fastly.md

```diff
@@ -173,6 +171,7 @@ Check with your [DNS registrar](https://lookup.icann.org/) for information about
 -  Provide a list of top-level and subdomains for each environment to your Customer Technical Advisor (CTA) or submit them in a Magento Support ticket.
 -  Wait for confirmation that the specified domains have been added to your Cloud environments.
 -  On Starter projects, add the domains to your Fastly service configuration. See [Manage domains]({{ site.baseurl }}/cloud/cdn/configure-fastly-customize-cache.html#manage-domains).
+-  For information about updating the DNS configuration, check with your [DNS registrar](https://lookup.icann.org/) for information about updating the DNS configuration.
```

Comment:
The phrase "For information about updating the DNS configuration" is repeated in the same sentence.

```suggestion
-  For information about updating the DNS configuration, check with your [DNS registrar](https://lookup.icann.org/) on the correct method for your domain service.
```

Reply:
I realized this mistake before seeing your suggestion.  Please review the updated version. Thx.